### DISCUSSION TITLE
Pie chart annotation

### DISCUSSION THREAD

**Discussion by renjiie** [airbnb/visx#1552] [General]
Hi,
I would like to display the percentage of each pie next to the pie arcs and also the legends below the pie chart something like the below image.
![demo_129_none-1](https://user-images.githubusercontent.com/37571081/182515229-288ef8d5-28fb-4d3f-8795-82fbff5bebe1.png)

 I have used @vx/annotation to display the percentage values but it is not rendering properly. I have attached the codesandbox link 
https://codesandbox.io/s/compassionate-liskov-njbkzj

**Comment by williaster** [comment]
converting this to a discussion since it's about usage, not a visx issue

**Reply by renjiie** [reply]
Is there any example to demo the usage of LinePathAnnotation from @vx? The components are different in @visx

**Reply by williaster** [reply]
`vx` is deprecated and no longer maintained. it graduated to `visx` which is all we support currently, I'd recommend upgrading to use `visx` as it's more type safe and has years of improvements.
